examples/benchmark/scene_test_1.py

```diff
@@ -143,6 +144,7 @@ def on_mouse_move(self, event):
             self.update()
 
     def on_mouse_wheel(self, event):
+        prof = Profiler()  # noqa
```

Comment:
... although then you can't later have `prof(...)` calls like you do. So never mind I guess.